src/js/__test__/CheckBox.test.tsx

```diff
@@ -38,6 +38,14 @@ describe('render IOS <Checkbox />', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders disabled IOS Checkbox', () => {
```

Comment:
please add tests for `disable={false}` and `none disable`

Reply:
will do. thx for the review

Reply:
updated @nicholaslee119, kindly check again.